internal/testrunner/testrunner.go

```diff
@@ -1,9 +1,21 @@
 package testrunner
 
 import (
+	"database/sql"
+
 	"github.com/dogmatiq/example"
 	"github.com/dogmatiq/testkit"
 )
 
-// Runner is a test runner for the example app.
+// Runner is a test runner for the example application, without projections.
 var Runner = testkit.New(&example.App{})
+
+// New returns a test runner for the example application.
+func New(db *sql.DB) *testkit.Runner {
```

Comment:
This should be updated to accept a `nil` value for `db`, see the comment on `testrunner.Runner` variable for more info.

Reply:
As discussed, this will be done in a separate PR. I created issue #36